app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files = command.split()
```

Comment:
First of all: Checklist was not readed
https://github.com/neostyle88/py-copy-file/blob/develop/checklist.md

1. Files is not correct name.
2. Check that commands.split() have exactly 3 params
3. Than you can use unzipping:
cmd, sourse_path, destination_path = command.split()
4. Check that cmd is equal to cp
5. Before opening files you can check that names are not the same. Not need to do that inside context manager